sw/device/silicon_creator/lib/ownership/isfb.h

```diff
@@ -0,0 +1,51 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_OWNERSHIP_ECDSA_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_OWNERSHIP_ECDSA_H_
+
+#include <stdint.h>
+
+#include "sw/device/silicon_creator/lib/error.h"
+#include "sw/device/silicon_creator/lib/manifest.h"
+#include "sw/device/silicon_creator/lib/ownership/owner_block.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Processes the ISFB boot request.
```

Comment:
Pardon my ignorance, but what is ISFB. Please explain it here and maybe in the patch description too.

Reply:
Updated commit description as well as stringdoc. 

Done.